### ISSUE TITLE
False positive SlevomatCodingStandard.Namespaces.UnusedUses.MismatchingCaseSensitivity

### ISSUE THREAD

**Issue by lptn** [slevomat/coding-standard#1257]
**version**: 7.0.13

**snippet**:
```php
<?php declare(strict_types=1);

namespace App\Http\Controllers\Users\Admin;

use App\Services\Payment\Reports\ReportDataBuilder;
use App\Models\Membership\Type\MembershipType;

final class AdminMemberLifespanReportController
{
    public function show(MembershipType $membershipType)
    {
        return (new ReportDataBuilder())
            ->title("Lifetime of {$membershipType::getName()} Users")
            ->generate()
    }
}
```
**output**:
```
ERROR | Case of reference name "membershipType" and use statement "MembershipType" does not match.
 | (SlevomatCodingStandard.Namespaces.UnusedUses.MismatchingCaseSensitivity)
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/db243812192ab2308098dfeec5ea8a95b93aba18
